descartes_planner/src/planning_graph.cpp

```diff
@@ -41,7 +41,7 @@ using namespace descartes_trajectory;
 namespace descartes_planner
 {
 
-const double MAX_JOINT_DIFF = M_PI;
+const double MAX_JOINT_DIFF = M_PI / 20;
```

Comment:
@ClayFlannigan Absolutely. The other half of the work I did was toward what you speak of. The user associates each point with a time window that indicates the time for the robot to realize this point relative to its previous position. 

I added a function to descarte's RobotModel interface that takes as arguments a pair of joint positions (source and target) and a time value and it returns true if the motion is possible in the time stated. This is used to do pruning instead of the existing hard-coded solution.

I put this code up seperately because it is pretty short and sweet. The timing stuff will probably take some iteration before everyone is happy.